install_instructions/client/raspberry-pi.md

```diff
@@ -1,67 +1,71 @@
 ---
-lastUpdated: "2021-04-06 12:11:00"
+lastUpdated: "2021-05-03 12:25:00"
 streamPiVersion: "1.0.0"
-editedBy: "SamuelQuinones"
+editedBy: "rnayabed, SamuelQuinones"
```

Comment:
editedBy should only ever be one user, this way we can track changes back to a specific user